src/tools/run-make-support/src/fs_wrapper.rs

```diff
@@ -0,0 +1,98 @@
+use std::fs;
+use std::path::Path;
+
+/// A safe wrapper around `std::fs::remove_file` that prints the file path if an operation fails.
```

Comment:
Suggestions:

- Not "safe" (that has a very specific meaning in Rust), but rather something more like "panic-on-fail" and "includes the file path in the panic message".
- You can inline doc link ``[`std::fs::remove_file`]``.